<!doctype html>
<!--[if lt IE 7]>		<html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en-GB"> <![endif]-->
<!--[if IE 7]>			<html class="no-js lt-ie9 lt-ie8" lang="en-GB"> <![endif]-->
<!--[if IE 8]>			<html class="no-js lt-ie9" lang="en-GB"> <![endif]-->
<!--[if gt IE 8]><!-->	<html class="no-js" lang="en-GB"> <!--<![endif]-->
<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<title>EuroSciPy</title>
	<link rel="shortcut icon" type="image/png" href="../theme_2020/images/favicon.ico"/>
	<meta name="title" content="EuroSciPy 2023 - Spotlight and Poster&nbsp;Session">
	<meta name="description" content="">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="../theme_2020/css/bootstrap.min.css">
	<link rel="stylesheet" href="../theme_2020/css/normalize.css">
	<link rel="stylesheet" href="../theme_2020/css/font-awesome.min.css">
	<link rel="stylesheet" href="../theme_2020/css/icomoon.css">
	<link rel="stylesheet" href="../theme_2020/css/customScrollbar.css">
	<link rel="stylesheet" href="../theme_2020/css/animate.css">
	<link rel="stylesheet" href="../theme_2020/css/owl.carousel.css">
	<link rel="stylesheet" href="../theme_2020/css/prettyPhoto.css">
	<link rel="stylesheet" href="../theme_2020/css/jquery.fullpage.css">
	<link rel="stylesheet" href="../theme_2020/css/transitions.css">
	<link rel="stylesheet" href="../theme_2020/css/main.css">
	<link rel="stylesheet" href="../theme_2020/css/color.css">
	<link rel="stylesheet" href="../theme_2020/css/responsive.css">
	<script src="../theme_2020/js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
	<style>
		html,
		body{height: auto;}
	</style>
</head>
<body class="tg-home tg-homeone">
	<!--[if lt IE 8]>
		<p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
	<![endif]-->
	<!--************************************
			Wrapper Start
	*************************************-->
	<div id="tg-wrapper" class="tg-wrapper tg-haslayout">
		<!--************************************
				Header Start
		*************************************-->
		<header id="tg-header" class="tg-header tg-headervtwo tg-haslayout">
			<div class="container-fluid">
				<div class="row">
					<div class="tg-logo col-xs-3"><a href="index.html"><img src="../static/2023/logo.png" alt="EuroSciPy logo"></a></div>
					<nav id="tg-nav" class="tg-nav pull-right">
						<div class="navbar-header">
							<button
								type="button"
								class="navbar-toggle collapsed"
								data-toggle="collapse"
								data-target="#tg-navigation"
								aria-expanded="false"
							>
								<span class="sr-only">Toggle navigation</span>
								<span class="icon-bar"></span>
								<span class="icon-bar"></span>
								<span class="icon-bar"></span>
							</button>
						</div>
						<div id="tg-navigation" class="collapse navbar-collapse tg-navigation">
							<ul>
								<li>
									<a href="index.html">Home</a>
								</li>
								<li>
									<a href="program_menu.html">Program</a>
										<ul class="sub-menu">
												<li><a href="program.html">Call for Proposals</a></li>
												<li><a href="schedule.html">Schedule</a></li>
												<li><a href="keynotes.html">Keynotes</a></li>
												<li><a href="poster_session.html">Spotlights and Posters</a></li>
												<li><a href="sprint.html">Sprint</a></li>
												<li><a href="social_event.html">Social Event</a></li>
												<li><a href="walking_tours.html">Walking Tours</a></li>
										</ul>
								</li>
								<li>
									<a href="maintainers.html">Maintainers</a>
								</li>
								<li>
									<a href="tickets.html">Registration</a>
								</li>
								<li>
									<a href="sponsors.html">Sponsors</a>
										<ul class="sub-menu">
												<li><a href="sponsoring.html">Become a Sponsor</a></li>
												<li><a href="current_sponsors.html">Current Sponsors</a></li>
												<li><a href="sponsoring-terms.html">Terms & Conditions</a></li>
										</ul>
								</li>
								<li>
									<a href="info.html">Info and Support</a>
										<ul class="sub-menu">
												<li><a href="about.html">About EuroSciPy</a></li>
												<li><a href="venue.html">Venue</a></li>
												<li><a href="finaid.html">Financial Aid</a></li>
												<li><a href="childcare.html">Child Care</a></li>
												<li><a href="code_of_conduct.html">Code of Conduct</a></li>
												<li><a href="organizers.html">Organizers</a></li>
												<li><a href="contact_us.html">Contact Us</a></li>
										</ul>
								</li>
							</ul>
						</div>
					</nav>
				</div>
			</div>
		</header>
		<!--************************************
				Header End
		*************************************-->
		<!--************************************
				Main Start
		*************************************-->
		<main id="tg-main" class="tg-main tg-haslayout">

<section class="tg-haslayout" id="euroscipy-page">
	<div class="container">
		<div class="row">
			<div class="tg-shortcode tg-aboutusshortcode">
				<div class="col-xs-8 col-md-offset-2 col-lg-offset-2 pull-left">
					<p><h1 id="spotlight-and-poster-session">Spotlight and Poster&nbsp;Session</h1>
<p>On Wednesday, a poster session is organized from 4.15 pm to 5.30 pm.
It will be preceded by a spotlight session taking place in Aula&nbsp;room.</p>
<p>You can find below the list of the&nbsp;poster.</p>
<h2 id="predictive-survival-analysis-and-competing-risk-modeling-with-gradient-boosted-cif">Predictive survival analysis and competing risk modeling with Gradient Boosted <span class="caps">CIF</span></h2>
<p><em>Vincent&nbsp;Maladiere</em></p>
<p><strong>Abstract:</strong> This tutorial is an introduction to our new survival analysis estimator called Gradient Boosted <span class="caps">CIF</span>. This model predicts cumulative incidence functions (<span class="caps">CIF</span>) in competing risk settings by leveraging scikit-learn&#8217;s&nbsp;HistGradientBoostingClassifier.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/8NHQGJ/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="pybamm-an-open-source-python-package-for-battery-simulation-contributes-to-a-solid-understanding-of-lithium-ion-batteries-degradation">PyBAMM, an open-source Python package for battery simulation, contributes to a solid understanding of lithium-ion batteries&nbsp;degradation</h2>
<p><em>Maria&nbsp;Kaninia</em></p>
<p><strong>Abstract:</strong> Being able to predict the degradation (capacity and power fade) of lithium-ion batteries is critical for the procurement and asset management of batteries. Often, this exercise is undertaken using empirical or semi-empirical models, which blend out the degradation mechanisms in favour or computational speed and&nbsp;simplicity.</p>
<p>By using PyBaMM (an open source package for battery simulation), we build bottom-up understanding of the discrete ageing mechanisms that impact battery State-of-Health indicators. In addition, PyBaMM comes complete with validated sets of parameters for different chemistries, which accelerates the deployment of the&nbsp;model.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/A8BJLV/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="scicode-widgets-a-toolkit-to-bring-computational-thinking-into-the-classroom">scicode-widgets: A toolkit to bring computational thinking into the&nbsp;classroom</h2>
<p><em>Alexander&nbsp;Goscinski</em></p>
<p><strong>Abstract:</strong> We introduce scicode-widgets, a Python package designed for educational purposes that facilitates the creation of interactive exercises for students in interdisciplinary fields of computational sciences. The implementation of computational experiments often demands extensive coding, hindering students&#8217; ability to effectively learn the interwork of coding experiments and analyzing their results. To reduce this workload for students, instructors can already provide a codebase and demand educational contributions from students. These contributions have to be embedded into a general workflow that involves coding experiments and analyzing their results. For that task scicode-widget provides the tools to connect custom pre- and postprocessing of students’ code, with the ability to verify the solution and to pass it to interactive visualizations driven by jupyter widgets. In the talk we demonstrate an example from a materials science class for teaching atomistic modeling and discuss a potential direction to integrate with&nbsp;nbgrader.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/ALH9X8/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="molgri-grids-for-molecular-dynamics">Molgri: grids for molecular&nbsp;dynamics</h2>
<p><em>Hana&nbsp;Zupan</em></p>
<p><strong>Abstract:</strong> Computational chemistry studies how molecules approach and interact at atomic precision and femtosecond timescales, a level of detail that is difficult to achieve in experiments. This enables us to understand important processes like the binding of drugs to&nbsp;receptors.</p>
<p>However, especially the first phase of binding – the approach and guidance towards a binding site - is difficult to study with the usual molecular dynamics approach. In the python package molgri, we instead create two sets of grids to study binding pathways. A grid of approach directions is generated based on a subdivision of an icosahedron; a grid of approach orientations based on a division of a&nbsp;half-hypercube.</p>
<p>After the user provides two molecular structures and the desired level of discretization, the molgri package generates a sequence of structures (a pseudo-trajectory) featuring all combinations of approach directions, distances and orientations. The output is immediately compatible with chemical software like <span class="caps">GROMACS</span> and openMM, so that this grid-based approach can be used as a starting point for simulations or used on its own to study long-range interactions guiding ligands to binding sites and to extract simulation-free models of kinetics (rate&nbsp;matrix).</p>
<p>The ease of use is taken into account during development; the generation of pseudo-trajectories can be easily done as a single command-line instruction, with examples provided in&nbsp;documentation.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/FQSXE7/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="streamlining-geospatial-machine-learning-with-srai">Streamlining Geospatial Machine Learning with <span class="caps">SRAI</span></h2>
<p><em>Szymon Woźniak, Piotr Szymański, Piotr Gramacki, Kamil Raczycki, Kacper&nbsp;Leśniara</em></p>
<p><strong>Abstract:</strong> This talk will introduce the audience to the Spatial Representations for Artificial Intelligence (srai), a Python library that provides an efficient, unified, and easy-to-use approach to geospatial problems. Attendees will get insights into the key functionalities and use cases of srai with a focus on how it enhances the Python scientific stack and encourages the application of Python in scientific and industrial geospatial&nbsp;research.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/GRHSGN/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="scipyjava-with-graalpy">SciPy❤Java with&nbsp;GraalPy</h2>
<p><em>Tim Felgentreff, Cosmin&nbsp;Basca</em></p>
<p><strong>Abstract:</strong> GraalPy brings up to date data science packages from Python to Java. Whether
analyzing data from Java enterprise applications or embedding Python
visualizations in Java&#8217;s cross-platform UIs, GraalPy makes it easier than ever.
In this talk we will show you how to get GraalPy from Conda or Pyenv, how it
performs on scientific Python workloads, and how to easily set up a
multi-language application with Java, Pygal, and&nbsp;SciPy.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/GSKSQJ/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="rna-seq-data-normalization-in-python"><span class="caps">RNA</span>-seq data normalization in&nbsp;Python</h2>
<p><em>Jure&nbsp;Zmrzlikar</em></p>
<p><strong>Abstract:</strong> Gene expression is commonly measured using <span class="caps">RNA</span> sequencing. In a nutshell, bioinformatic analysis includes aligning sequencing reads to a reference genome and counting expression as the number of reads per gene or transcript. Normalization of gene counts is important for accurate interpretation and downstream analysis, and the widely used methods like <span class="caps">TPM</span> and <span class="caps">RPKM</span> make us believe that <span class="caps">RNA</span>-seq normalization is a solved problem. This poster will present some alternatives and discuss why there may be more suitable ways of transforming gene expression counts for machine learning&nbsp;applications.</p>
<p>We developed <a href="https://github.com/genialis/rnanorm">RNAnorm</a>, an <span class="caps">RNA</span>-seq data normalization toolkit in Python. This open-source package (re-)implements a range of normalization methods including <span class="caps">CPM</span>, <span class="caps">FPKM</span>, <span class="caps">TPM</span>, <span class="caps">TMM</span>, <span class="caps">UQ</span>, <span class="caps">CUF</span>, and <span class="caps">CTF</span>. The code architecture is compatible with scikit-learn’s fit and transform interface, allowing users to rapidly prototype normalization strategies and assess their impact on model&nbsp;performance.</p>
<p>The poster will present the theoretical background of the methods from the perspective of the bias they address (i.e. library size, gene length, <span class="caps">RNA</span> composition) and their respective strengths and weaknesses. It will also showcase the usage od RNAnorm package to perform various normalization tasks: from simple one-line normalization to a more complex grid search to find the best combination of normalizations for a given machine learning&nbsp;problem.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/JBUYJL/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="scverse-foundational-tools-for-single-cell-omics-analysis">scverse: Foundational tools for single-cell omics&nbsp;analysis</h2>
<p><em>Isaac&nbsp;Virshup</em></p>
<p><strong>Abstract:</strong> scverse (scverse.org) is a consortium of core Python tools for storage and analysis of high-dimensional single-cell data. These tools provide scalable infrastructure for hundreds of dependent packages and have been used to study the human immune system, aging, and <span class="caps">COVID</span>-19. The consortium provides a continued maintenance plan, community platforms, and organization for the hubs of our research software ecosystem. We will introduce scverse and present ongoing work addressing scalability, spatial omics, and community&nbsp;building.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/NPNXNA/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="runtime-control-of-numerical-groundwater-models">Runtime-Control of Numerical Groundwater&nbsp;Models</h2>
<p><em>Mike Müller, Lúcia&nbsp;Pedrosa</em></p>
<p><strong>Abstract:</strong> The widely-used finite volume groundwater flow and transport model <span class="caps">MODLFOW</span> 6 is implemented in Fortran. The typical work flow consists of the steps (1) pre-processing on input data, (2) running the model, and (3) post-processing of model results. For a variety of problems this workflow has to be applied repeatedly, modifying values of input data based on modeling results  The here presented tool <code>pymf6</code> enables the model user to access intermediate model results at runtime and modifying model variables with Python. This allows to greatly reduce the number of model runs. Often, one model run can replace many manual iterations, leading to a much faster workflow. Furthermore, complex tasks that would not be feasibly with the manual workflow can be solved. <code>pymf6</code> can also be used to couple <span class="caps">MODLFOW</span> 6 with other numerical or analytical&nbsp;models.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/P39VK3/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="automating-data-collection-and-analysis-in-citizen-science-projects">Automating data collection and analysis in citizen science&nbsp;projects</h2>
<p><em>Dimitrios&nbsp;Ladakis</em></p>
<p><strong>Abstract:</strong> Citizen science is a powerful tool used by scientists in a wide range of fields to aid the utilisation of scientific data, that would otherwise take a very long time to process and analyse, while being an important way to engage with the public. <span class="caps">RFI</span> has a number of citizen science projects, under the name ‘Science Scribbler’, for collecting annotation data from biological 3D imaging techniques. Annotated data can be used to guide segmentation from images in order to understand morphological features at the cellular level in health and disease. To support current citizen science projects, and to facilitate planning of future ones, automating the way images are generated and annotation data is processed is of major importance. A web tool for on-the-fly image generation from 3D volumes of data is being created to provide an easy way to generate images for annotation by volunteers via the Zooniverse citizen science platform, but also to allow minimisation of storage space requirements. Furthermore a library for automating data processing and analysis is in construction that can be used as a pipeline to extract, pool, visualise, filter and prepare annotation data for segmentation and other types of&nbsp;analysis.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/P9E7HX/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="writing-tests-for-your-containers-with-container-canary">Writing tests for your containers with Container&nbsp;Canary</h2>
<p><em>Jacob&nbsp;Tomlinson</em></p>
<p><strong>Abstract:</strong> When we write code we also write tests to ensure that our software functions as expected. But with containers, it is far more common to document assumptions and leave it at&nbsp;that.</p>
<p>Containers are the building blocks of modern compute platforms and they have interfaces like any other software. The software they contain, commands they run and ports they listen on all make up that interface and when composed into larger systems many assumptions are made about how containers will behave. So we should test&nbsp;that!</p>
<p>Container Canary is a black-box testing tool for container images and in this talk, we will talk about why you should test your containers, how to write canary test manifests and how to use the canary <span class="caps">CLI</span> tool to run&nbsp;them.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/RLJETB/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="skrub-preparing-tables-for-machine-learning">skrub: preparing tables for machine&nbsp;learning</h2>
<p><em>Jovan&nbsp;Stojanovic</em></p>
<p><strong>Abstract:</strong> Are you wondering what to do with all of the tables you have?
And how to make sense of all of the dirty&nbsp;data?</p>
<p>Data science is increasingly faced with multiple-sourced, non-standardized dirty&nbsp;tables.</p>
<p>We have what you need: skrub is a Python package for preparing your dirty tables for machine&nbsp;learning!</p>
<p>dirty-cat evolves into skrub and gives you the right tools to clean, transform and join your tables for machine&nbsp;learning.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/SCLBPU/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="from-ab-initio-to-scattering-experiments-a-python-workflow-for-constructing-neuroevolution-potentials">From ab-initio to scattering experiments: A Python workflow for constructing neuroevolution&nbsp;potentials</h2>
<p><em>Eric&nbsp;Lindgren</em></p>
<p><strong>Abstract:</strong> Computer simulations such as Molecular Dynamics (<span class="caps">MD</span>) are an effective tool in studying a broad range of structural and dynamical properties in atomic-scale systems, but the connection to X-ray or neutron scattering experiments is not always straightforward.
In this work, we present a Python workflow that starts with the construction of machine learned potentials and ends with the prediction of experimental observables, such as the dynamical structure factor.
Specifically, the Python package <code>Calorine</code> is used to train neuro-evolution potentials (<span class="caps">NEP</span>) with ab-initio level accuracy using the software package <code>GPUMD</code>, whilst still retaining a familiar interface based around the Python framework <code>ASE</code>.
The trained potentials are then used to run <span class="caps">MD</span> within <span class="caps">GPUMD</span>, also done through <code>Calorine</code>.
Finally, the resulting <span class="caps">MD</span> trajectories are analyzed with the Python package <code>Dynasor</code> in order to calculate dynamical structure factors, which in turn may be convoluted with instrument parameters in order to compare to experiments.
We apply our workflow to crystalline benzene as a prototype system, but the workflow may be readily extended to different&nbsp;systems.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/XPHSMW/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="mlees-an-initial-guide-through-the-thickets-of-machine-learning-research-in-earth-and-environmental-sciences"><span class="caps">MLEES</span> - An Initial Guide through the Thickets of Machine Learning Research in Earth and Environmental&nbsp;Sciences</h2>
<p><em>Milton&nbsp;Gomez</em></p>
<p><strong>Abstract:</strong> Students in earth and environmental sciences generally have limited time dedicated to learning programming. As machine learning (<span class="caps">ML</span>) techniques become more widespread in our fields, we find that providing course materials that introduce the concepts behind <span class="caps">ML</span> techniques and materials showcasing the use of techniques in recent literature helps students develop the skills necessary to apply <span class="caps">ML</span> to their own research. We tackle each <span class="caps">ML</span> concept through notebooks that use benchmark datasets and take inspiration from renowned references. We then focus on reproduction of the work in over 5 papers/preprints that successfully applied <span class="caps">ML</span> to climate science, emphasizing the importance of reproducibility and establishing trust in <span class="caps">ML</span> research. We also provide a platform to highlight the result of our students&#8217; work via final projects in which they apply the course’s concepts in their Master&#8217;s research. These further show how the course promotes work at the intersection of environmental science and elementary <span class="caps">ML</span> methods. The materials are open source, hosted on GitHub, and further developments will increase the variety in deep learning algorithms presented throughout the course. We also provide materials covering an introduction to the Python skills required to follow the notebooks.  Finally, part of the materials have already been used in a Massively Open Online Course (<span class="caps">MOOC</span>) developed by <span class="caps">ECMWF</span>, we hope to continue with development of a more robust Jupyter book and to develop our own <span class="caps">MOOC</span>.</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/YQNNB7/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2 id="streamline-your-scientific-work-with-automl">Streamline Your Scientific Work with&nbsp;AutoML</h2>
<p><em>Piotr Płoński, Aleksandra&nbsp;Płońska</em></p>
<p><strong>Abstract:</strong> How can AutoML technology be used in scientific research? Take a sneak peek at where your data analysis can be easier with mljar-supervised. By automating every step of the pipeline, from preprocessing to algorithm selection and explanation, researchers can easily advance their work. We will showcase the various fields in which scientists have implemented AutoML&nbsp;technology</p>
<p><a href="https://pretalx.com/euroscipy-2023/talk/ZMLBES/"><button type="button" class="btn btn-primary">All details</button></a></p></p>
				</div>
			</div>
		</div>
	</div>
</section>
		</main>
		<!--************************************
				Main End
		*************************************-->

		<!--************************************
				Footer Start
		*************************************-->
		<footer id="tg-footer" class="tg-footer tg-haslayout tg-footervthree">
			<!-- <div class="tg-sectionspace tg-haslayout tg-bgtwitter">
				<div class="container">
					<div class="row">
						<div class="col-xs-12 col-sm-12 col-xs-offset-0 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2">
							<div class="tg-twitterpost">
								<i class="fa fa-twitter" style="color:#00ACEE;"></i>
								<a class="twitter-timeline" data-lang="en" data-height="400" data-dnt="true" href="https://twitter.com/euroscipy">Tweets by EuroSciPy</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
							</div>
						</div>
					</div>
				</div>
				col-xs-12 col-sm-12 col-md-12 col-lg-12
			</div> -->

			<div class="tg-foorterbar container-fluid">
				<div class="row">
					<div class="col-sm-5 col-sm-offset-1 tg-copyrights">
						2025 All Rights Reserved ©&nbsp;EuroSciPy
					</div>
					<div class="col-sm-2 tg-logo"><a href="index.html">
						<img
							src="../static/2023/logo.png"
							alt="EuroSciPy logo"
						/>
					</a></div>
					<div class="col-sm-2 col-sm-offset-1 tg-socialicons"><ul>
						<li><a href="https://twitter.com/intent/follow?original_referer=https%3A%2F%2Fwww.euroscipy.org%2F2019%2F&ref_src=twsrc%5Etfw&region=follow_link&screen_name=euroscipy&tw_p=followbutton"><i class="fa fa-twitter"></i></a></li>
						<li><a href="https://t.me/euroscipy"><i class="fa fa-telegram"></i></a></li>
						<li><a href="mailto:info@euroscipy.org"><i class="fa fa-envelope "></i></a></li>
					</ul></div>
				</div>
			</div>
		</footer>
		<!--************************************
				Footer End
		*************************************-->	</div>
	<!--************************************
			Wrapper End
	*************************************-->
	<script src="../theme_2020/js/vendor/jquery-library.js"></script>
	<script src="../theme_2020/js/vendor/bootstrap.min.js"></script>
	<script src="../theme_2020/js/customScrollbar.min.js"></script>
	<script src="../theme_2020/js/packery.pkgd.min.js"></script>
	<script src="../theme_2020/js/owl.carousel.min.js"></script>
	<script src="../theme_2020/js/jquery.hoverdir.js"></script>
	<script src="../theme_2020/js/jquery.vide.min.js"></script>
	<script src="../theme_2020/js/jquery.fullpage.js"></script>
	<script src="../theme_2020/js/isotope.pkgd.js"></script>
	<script src="../theme_2020/js/prettyPhoto.js"></script>
	<script src="../theme_2020/js/countdown.js"></script>
	<script src="../theme_2020/js/countTo.js"></script>
	<script src="../theme_2020/js/appear.js"></script>
	<script src="../theme_2020/js/gmap3.js"></script>
	<script src="../theme_2020/js/main.js"></script>
</body>
</html>