<!doctype html>
<!--[if lt IE 7]>		<html class="no-js lt-ie9 lt-ie8 lt-ie7" lang="en-GB"> <![endif]-->
<!--[if IE 7]>			<html class="no-js lt-ie9 lt-ie8" lang="en-GB"> <![endif]-->
<!--[if IE 8]>			<html class="no-js lt-ie9" lang="en-GB"> <![endif]-->
<!--[if gt IE 8]><!-->	<html class="no-js" lang="en-GB"> <!--<![endif]-->
<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<title>EuroSciPy</title>
	<link rel="shortcut icon" type="image/png" href="../theme_2020/images/favicon.ico"/>
	<meta name="title" content="EuroSciPy 2022 - Spotlight and Poster&nbsp;Session">
	<meta name="description" content="">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="../theme_2020/css/bootstrap.min.css">
	<link rel="stylesheet" href="../theme_2020/css/normalize.css">
	<link rel="stylesheet" href="../theme_2020/css/font-awesome.min.css">
	<link rel="stylesheet" href="../theme_2020/css/icomoon.css">
	<link rel="stylesheet" href="../theme_2020/css/customScrollbar.css">
	<link rel="stylesheet" href="../theme_2020/css/animate.css">
	<link rel="stylesheet" href="../theme_2020/css/owl.carousel.css">
	<link rel="stylesheet" href="../theme_2020/css/prettyPhoto.css">
	<link rel="stylesheet" href="../theme_2020/css/jquery.fullpage.css">
	<link rel="stylesheet" href="../theme_2020/css/transitions.css">
	<link rel="stylesheet" href="../theme_2020/css/main.css">
	<link rel="stylesheet" href="../theme_2020/css/color.css">
	<link rel="stylesheet" href="../theme_2020/css/responsive.css">
	<script src="../theme_2020/js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
	<script src="https://pretalx.com/euroscipy-2022/schedule/widget/v2.en.js"></script>
	<style>
		html,
		body{height: auto;}
	</style>
</head>
<body class="tg-home tg-homeone">
	<!--[if lt IE 8]>
		<p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
	<![endif]-->
	<!--************************************
			Wrapper Start
	*************************************-->
	<div id="tg-wrapper" class="tg-wrapper tg-haslayout">
		<!--************************************
				Header Start
		*************************************-->
		<header id="tg-header" class="tg-header tg-headervtwo tg-haslayout">
			<div class="container-fluid">
				<div class="row">
						<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
							<strong class="tg-logo"><a href="index.html"><img src="../static/2022/logo.png" alt="EuroSciPy Bilbao 2020 logo"></a></strong>
							<nav id="tg-nav" class="tg-nav">
								<div class="navbar-header">
									<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#tg-navigation" aria-expanded="false">
										<span class="sr-only">Toggle navigation</span>
										<span class="icon-bar"></span>
										<span class="icon-bar"></span>
										<span class="icon-bar"></span>
									</button>
								</div>
								<div id="tg-navigation" class="collapse navbar-collapse tg-navigation">
									<ul>
										<li>
											<a href="index.html">Home</a>
										</li>
										<li>
											<a href="program_menu.html">Program</a>
												<ul class="sub-menu">
														<li><a href="program.html">Call for Proposals</a></li>
														<li><a href="https://pretalx.com/euroscipy-2022/schedule/">Schedule</a></li>
														<li><a href="keynotes.html">Keynotes</a></li>
														<li><a href="poster_session.html">Spotlights and Posters</a></li>
														<li><a href="social_event.html">Social Event</a></li>
														<li><a href="sprint.html">Sprint</a></li>
														<li><a href="pyladies_lunch.html">PyLadies Lunch</a></li>
												</ul>
										</li>
										<li>
											<a href="maintainers.html">Maintainers</a>
										</li>
										<li>
											<a href="tickets.html">Registration</a>
										</li>
										<li>
											<a href="sponsors.html">Sponsors</a>
												<ul class="sub-menu">
														<li><a href="sponsoring.html">Become a Sponsor</a></li>
														<li><a href="current_sponsors.html">Current Sponsors</a></li>
														<li><a href="sponsoring-terms.html">Terms & Conditions</a></li>
												</ul>
										</li>
										<li>
											<a href="info.html">Info and Support</a>
												<ul class="sub-menu">
														<li><a href="venue.html">Venue</a></li>
														<li><a href="finaid.html">Financial Aid</a></li>
														<li><a href="childcare.html">Child Care</a></li>
														<li><a href="code_of_conduct.html">Code of Conduct</a></li>
														<li><a href="contact_us.html">Contact Us</a></li>
														<li><a href="about.html">Organizers</a></li>
														<li><a href="covid_policy.html">COVID Policy</a></li>
												</ul>
										</li>
									</ul>
								</div>
							</nav>
						</div>
					</div>
			</div>
		</header>
		<!--************************************
				Header End
		*************************************-->
		<!--************************************
				Main Start
		*************************************-->
		<main id="tg-main" class="tg-main tg-haslayout">

<section class="tg-sectionspace tg-haslayout" id="euroscipy-page">
	<div class="container">
		<div class="row">
			<div class="tg-shortcode tg-aboutusshortcode">
				<div class="col-xs-8 col-sm-8 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2 pull-left">
					<p><h1>Spotlight and Poster&nbsp;Session</h1>
<p>On Wednesday, a poster session is organized from 3.40 pm to 5 pm. During the
first 40 minutes, each poster presenter will give a 2-minutes spotlight
presentation. These presentations will be followed by a 40 minutes poster&nbsp;session.</p>
<p>You can find below the list of the&nbsp;poster.</p>
<h2>BioConvert: a comprehensive format converter for life&nbsp;sciences</h2>
<p><em>thomas&nbsp;cokelaer</em></p>
<p><strong>Abstract:</strong> Life science uses many different formats. They may be old, or with complex syntax and converting these formats may be challenging for scientists. Bioconvert aims to provide a standard tool/interface to convert life science data formats from one to&nbsp;another.</p>
<p>Many conversion tools already exist but they may be dispersed, focused on a few specific formats, difficult to install, or not optimised. With Bioconvert, we plan to cover a wide spectrum of format conversions; we will re-use existing tools when possible and provide an interface to compare different conversion tools or methods via benchmarking. New implementations are provided when considered better than existing&nbsp;ones.</p>
<p>Bioconvert is developed in Python using continuous integration, a test suite and extensive Sphinx documentation. In March 2022, we had 48 formats, 98 direct conversions (125 different&nbsp;methods).</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/7SNGMW/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>Sequana: a set of Next Generation Sequencing&nbsp;pipelines</h2>
<p><em>thomas&nbsp;cokelaer</em></p>
<p><strong>Abstract:</strong> Sequana software is developed within a Sequencing platform at Institut Pasteur. It provides a Python library dedicated to Next Generation Sequencing (<span class="caps">NGS</span>) analysis including visualisation of <span class="caps">NGS</span> formats.
Sequana is also a project that provides (i) a set of pipelines dedicated to <span class="caps">NGS</span> in the form of Snakefiles (Makefile-like with Python syntax based on Snakemake framework), (ii) tools to help in the creation of such pipelines, (iii) a graphical interface for Snakemake framework, (iv) standalone applications for <span class="caps">NGS</span> analysis. Pipelines can be run locally or on <span class="caps">HPC</span> clusters. Common user interface is provided to ease user interface. These <span class="caps">NGS</span> pipelines are ready for production and have been applied on hundreds of projects including Covid variant detection, genomic, transcriptomics,&nbsp;etc</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/8JT9QA/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>Conda Store : easy environments management <span class="amp">&amp;</span> reproducibility for Teams and&nbsp;Enterprises</h2>
<p><em>Pierre-Olivier&nbsp;Simonard</em></p>
<p><strong>Abstract:</strong> End users think in terms of environments, not packages. Conda Store makes it easy for data scientists to define their environments, ensures reproducibility, productionizing, easing collaboration, and reduces friction and latency between developers and <span class="caps">IT</span>.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/9QBKCE/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>JupyterLab 4 and the future of Jupyter&nbsp;Notebook</h2>
<p><em>Jeremy Tuloup, Frédéric&nbsp;Collonval</em></p>
<p><strong>Abstract:</strong> JupyterLab is a powerful computational environment for interactive data science in the browser, and the new version 4 release comes with many new features and&nbsp;improvements.</p>
<p>The Jupyter Notebook project decided to base its next major version 7 on JupyterLab components and extensions, which means many JupyterLab features are also available to Jupyter Notebook&nbsp;users.</p>
<p>In this presentation, we will demo all the features coming in these new versions and how users can seamlessly switch from one notebook interface to&nbsp;another.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/ALLCZ7/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>The Beauty of&nbsp;Zarr</h2>
<p><em>Sanket Verma, Jonathan&nbsp;Striebel</em></p>
<p><strong>Abstract:</strong> This poster showcases Zarr, an open-source data format for storing chunked, compressed N-dimensional arrays. We present a systematic approach to understanding and implementing Zarr by showing how it works and the need for using it. Zarr is based on an open technical specification, making implementations across several languages possible. The focus here is on Zarr’s Python implementation and its interoperability with the existing libraries in the PyData&nbsp;stack.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/APTJNF/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>Scipp plot: modular interactive plotting from graph&nbsp;nodes</h2>
<p><em>nvaytet</em></p>
<p><strong>Abstract:</strong> We present the plotting framework of the Scipp package (https://scipp.github.io/) for multi-dimensional arrays.
Based on the Model View Controller pattern, it uses a set of nodes connected in a graph to represent a sequence of processing steps that can be applied to the data before plotting it onto the figure axes.
A common example of this could be a 2D scatter plot, accompanied by 1D histograms of the data points on the top and right hand side of the scatter axes.
The histogramming nodes, that lie below the original root data node in the graph, perform a histogram operation in each of the X and Y dimensions, and their results get sent to the top and right plotting&nbsp;axes.</p>
<p>The use of a graph of connected nodes opens up the opportunity for a very modular way of creating interactive plots.
For instance, using a library of widgets (such as ipywidgets), we can change the input to one of the nodes, which notifies all the nodes below it about the change.
This means that modifying a parameter of the scatter data with e.g. a slider, would automatically update not only the main scatter plot, but also the histograms on the&nbsp;sides.</p>
<p>Any function (smoothing, fitting, filtering …) can be used inside a node, and any number of axes (or views) can be attached to a given node. This flexibility allows users to create complicated interactive visualizations with just a few lines of&nbsp;code.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/BAL88N/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2><span class="caps">CLAIMED</span> - An open source unified platform for batch, streaming and microservices based data&nbsp;science</h2>
<p><em>Romeo&nbsp;Kienzler</em></p>
<p><strong>Abstract:</strong> Data are processed in pipelines – either an entire data set, in batches or one by one. A variety of programming languages, frameworks and libraries exists. In <span class="caps">CLAIMED</span> – the component library for <span class="caps">AI</span>, Machine Learning, <span class="caps">ETL</span> and Data Science – we provide an opinionated set of coarse grained components implemented as jupyter notebooks. Through C3, the claimed component compiler those can be (as of now) transformed into Kubeflow Pipeline Components, Airflow Operators or simple (docker) container images to be executed on Knative. An adapter implemented as side car transforms those into either streaming components (currently http(s) and Kafka) or micro services – with scale to zero support. Using the jupyter lab Elyra pipeline editor and <span class="caps">CLAIMED</span>, anybody can create data science pipelines without programming skills. But the source code is only one click away. The jupyter notebook backing the component is available for review, adjustments or improvements of the&nbsp;components.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/BZDJXH/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>Python in&nbsp;Storm</h2>
<p><em>Soundharya&nbsp;Khanapur</em></p>
<p><strong>Abstract:</strong> The advancement and development in the field of Technology and Communication call for a need for Real-time data processing that is fast and fault-tolerant. Apache Storm provides an epoch platform to develop applications that can process a multitude of data in real-time. Being distributed, Storm is predominantly fast and maintains high accuracy with its topological analysis and task completion&nbsp;checks.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/GMBXCA/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>Informative and pleasant dataviz with Raincloud&nbsp;plot</h2>
<p><em>Davide&nbsp;Poggiali</em></p>
<p><strong>Abstract:</strong> Categorical plots offer a variety of plotting styles that allows the user to picture even large datasets showing some summary statistics of the data. In some case graphs can be misleading, either unwilling or on purpose. The reader can be confused or even get an incorrect idea of the phenomenon underlying the data.
In this talk we introduce the Raincloud plot, a multi-language plotting style aimed to create charming and informative graphical representations of a dataset. After some introduction, we will offer a simple tutorial that will cover different use cases. We will then compare the Raincloud plots with some other plot styles, showing that some data misunderstanding can be avoided with a sufficiently detailed&nbsp;plot.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/KWFH9N/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>Renku-Python: Reproducible and Reusable&nbsp;Workflows</h2>
<p><em>Ralf&nbsp;Grubenmann</em></p>
<p><strong>Abstract:</strong> Renku is a platform that bundles together various tools for reproducible and collaborative data analysis projects. Here we take a deep dive into the Python <span class="caps">CLI</span> and library component of the Renku platform, highlighting its functionality for recording and executing workflows both locally and remotely, as well as its architecture for storing recorded metadata in a knowledge graph and how this can be extended by third-party&nbsp;plugins.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/PFQPTF/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>dirty_cat : a Python package for Machine Learning on Dirty Categorical&nbsp;Data</h2>
<p><em>Lilian&nbsp;Boulard</em></p>
<p><strong>Abstract:</strong> In this talk, we will introduce &#8220;dirty_cat&#8221;, a Python library for encoding dirty, non-curated categorical features into numerical features while preserving similarities.
We will focus on a few methods implemented in the similarity encoder, the Gamma-Poisson encoder, the min-hash encoder and the&nbsp;super-vectorizer.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/PRYDKM/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>Scipp: Multi-dimensional data arrays with labeled dimensions for dense and binned&nbsp;data</h2>
<p><em>Simon&nbsp;Heybrock</em></p>
<p><strong>Abstract:</strong> Inspired by Xarray, Scipp (https://scipp.github.io/) enriches raw NumPy-like multi-dimensional arrays of data by adding named dimensions and associated coordinates. For an even more intuitive and less error-prone user experience, Scipp furthermore adds physical units to arrays and their coordinates. Scipp data arrays additionally support a dictionary of masks, basic propagation of uncertainties, and bin-edge&nbsp;coordinates.</p>
<p>On top of the above, Scipp&#8217;s key feature is support for multi-dimensional non-destructive binning of record-based &#8220;tabular&#8221; data into arrays of bins. The use of labeled arrays with coordinates to represent the table of records allows for clear conceptual association of a record&#8217;s metadata with dimensions and coordinates of the array of bins. Based on this, Scipp can provide fast, flexible, and efficient binning, rebinning, and filtering operations, all while preserving the original individual&nbsp;records.</p>
<p>Scipp ships with data display and visualization features for Jupyter notebooks, including a powerful plotting&nbsp;interface.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/RANUF3/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>ReservoirPy: Efficient Training of Recurrent Neural Networks for Timeseries&nbsp;Processing</h2>
<p><em>Xavier&nbsp;Hinaut</em></p>
<p><strong>Abstract:</strong> ReservoirPy is a simple user-friendly library based on Python scientific modules. It provides a flexible interface to implement efficient Reservoir Computing (<span class="caps">RC</span>) architectures with a particular focus on Echo State Networks (<span class="caps">ESN</span>). Advanced features of ReservoirPy allow to improve computation time efficiency on a simple laptop compared to basic Python implementation, with datasets of any&nbsp;size.</p>
<p>Some of its features are: offline and online training, parallel implementation, sparse matrix computation, fast spectral initialization, advanced learning rules (e.g. Intrinsic Plasticity) etc. It also makes possible to easily create complex architectures with multiple reservoirs (e.g. deep reservoirs), readouts, and complex feedback loops. Moreover, graphical tools are included to easily explore hyperparameters with the help of the hyperopt library. It includes several tutorials exploring exotic architectures and examples of scientific papers reproduction. Moreover, graphical tools are included to easily explore hyperparameters with the help of the hyperopt library. ReservoirPy is available on GitHub https://github.com/reservoirpy/reservoirpy with the open source <span class="caps">MIT</span> license, it includes a detailed documentation https://reservoirpy.readthedocs.io and a pypi package for easy&nbsp;installation.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/S7NHQE/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>pyLife – a python package for mechanical lifetime&nbsp;assessment</h2>
<p><em>Johannes&nbsp;Mueller</em></p>
<p><strong>Abstract:</strong> pyLife is a Python package covering state of the art algorithms of mechanical lifetime assessment and material fatigue. In this talk we will see a very quick glance of mechanical lifetime estimation and how we can combine classical methods from mechanical engineering with methods from data science. We will see how pyLife&#8217;s modules can be used to build versatile solutions for the engineer&#8217;s desktop as well as server based solutions for manufacturing and quality assurance with a high degree of automation. As pyLife is an Open Source project, everyone is welcome to collaborate. We are curious if we can establish a developer community in the realm of mechanical engineering. We are aiming especially towards university teachers using pyLife for teaching and research&nbsp;purposes.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/WBLY9B/"><button type="button" class="btn btn-primary">All details</button></a></p>
<h2>napari: a multi-dimensional image visualization, annotation, and analysis platform in&nbsp;Python</h2>
<p><em>Kevin&nbsp;Yamauchi</em></p>
<p><strong>Abstract:</strong> Napari is an interactive, <span class="caps">GPU</span>-accelerated, nD image viewer written in python. It displays images in a 2D or 3D canvas, then provides sliders for any additional dimensions in a dataset. It can also overlay associated data such as segmentations, points, polygons, surfaces, vectors, and tracks. Finally, napari is well-integrated with the scientific python ecosystem: NumPy arrays are the primary data structure used for visualization, and other standard arrays (such as Zarr or Dask arrays) are also supported. This makes it easy to insert interactive visualization, curation, and annotation steps into any workflow using standard SciPy libraries such as NumPy, SciPy, dask, and scikit-image. In this talk, I will introduce napari and demonstrate how napari can be used for interactive image&nbsp;analysis.</p>
<p><a href="https://pretalx.com/euroscipy-2022/talk/ZPLK87/"><button type="button" class="btn btn-primary">All details</button></a></p></p>
				</div>
			</div>
		</div>
	</div>
</section>
		</main>
		<!--************************************
				Main End
		*************************************-->

		<!--************************************
				Footer Start
		*************************************-->
		<footer id="tg-footer" class="tg-footer tg-haslayout tg-footervthree">
			<div class="tg-sectionspace tg-haslayout tg-bgtwitter">
				<div class="container">
					<div class="row">
						<div class="col-xs-12 col-sm-12 col-xs-offset-0 col-md-8 col-md-offset-2 col-lg-8 col-lg-offset-2">
							<div class="tg-twitterpost">
								<i class="fa fa-twitter" style="color:#00ACEE;"></i>
								<a class="twitter-timeline" data-lang="en" data-height="400" data-dnt="true" href="https://twitter.com/euroscipy">Tweets by EuroSciPy</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
							</div>
						</div>
					</div>
				</div>
			</div>


			<div class="tg-foorterbar">
				<div class="container">
					<div class="row">
						<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
							<p class="tg-copyrights">2022 All Rights Reserved © EuroSciPy</p>
							<ul class="tg-socialicons">
								<li><a href="https://twitter.com/intent/follow?original_referer=https%3A%2F%2Fwww.euroscipy.org%2F2019%2F&ref_src=twsrc%5Etfw&region=follow_link&screen_name=euroscipy&tw_p=followbutton"><i class="fa fa-twitter"></i></a></li>
								<li><a href="https://t.me/euroscipy"><i class="fa fa-telegram"></i></a></li>
								<li><a href="mailto:info@euroscipy.org"><i class="fa fa-envelope "></i></a></li>
							</ul>
							<strong class="tg-logo"><a href="index.html"><img src="../static/2022/logo.png" alt="EuroSciPy Bilbao 2020 logo"></a></strong>
						</div>
					</div>
				</div>
			</div>
		</footer>
		<!--************************************
				Footer End
		*************************************-->	</div>
	<!--************************************
			Wrapper End
	*************************************-->
	<script src="../theme_2020/js/vendor/jquery-library.js"></script>
	<script src="../theme_2020/js/vendor/bootstrap.min.js"></script>
	<script src="https://maps.google.com/maps/api/js?key=AIzaSyCR-KEWAVCn52mSdeVeTqZjtqbmVJyfSus&language=en"></script>
	<script src="../theme_2020/js/customScrollbar.min.js"></script>
	<script src="../theme_2020/js/packery.pkgd.min.js"></script>
	<script src="../theme_2020/js/owl.carousel.min.js"></script>
	<script src="../theme_2020/js/jquery.hoverdir.js"></script>
	<script src="../theme_2020/js/jquery.vide.min.js"></script>
	<script src="../theme_2020/js/jquery.fullpage.js"></script>
	<script src="../theme_2020/js/isotope.pkgd.js"></script>
	<script src="../theme_2020/js/prettyPhoto.js"></script>
	<script src="../theme_2020/js/countdown.js"></script>
	<script src="../theme_2020/js/countTo.js"></script>
	<script src="../theme_2020/js/appear.js"></script>
	<script src="../theme_2020/js/gmap3.js"></script>
	<script src="../theme_2020/js/main.js"></script>
</body>
</html>